### PR TITLE
Update protected browser storage for 5.0

### LINKED ISSUE #20350: Update protected browser storage for Blazor Server
The protected browser support was also moved into the shared framework in RC2, so you no longer need to add a separate package and the services are registered when you add the services for Blazor Server. You just need to add a `using` directive for `Microsoft.AspNetCore.Componets.Server.ProtectedBrowserStorage`.

https://github.com/dotnet/AspNetCore.Docs/pull/20140#discussion_r513979545

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ae14b3e3-3271-b919-9e0a-6f317a6b2d2d
* Version Independent ID: e5e1273b-195e-5da1-b4aa-66bbcff1425b
* Content: [ASP.NET Core Blazor state management](https://docs.microsoft.com/en-us/aspnet/core/blazor/state-management?view=aspnetcore-5.0&pivots=server#aspnet-core-protected-browser-storage)
* Content Source: [aspnetcore/blazor/state-management.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/state-management.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #20350

[Internal Review Topic (links to section)](https://review.docs.microsoft.com/en-us/aspnet/core/blazor/state-management?view=aspnetcore-3.1&branch=pr-en-us-20351&view=aspnetcore-5.0&pivots=server#aspnet-core-protected-browser-storage)

### CODE REVIEW

File: aspnetcore/blazor/state-management.md [hunk 1]
```diff
@@ -239,7 +230,7 @@ In any component that requires loading or saving data to browser storage, use th
 The choice depends on which browser storage location you wish to use. In the following example, `sessionStorage` is used:
 
 ```razor
-@using Microsoft.AspNetCore.Components.Web.Extensions
+@using Microsoft.AspNetCore.Componets.Server.ProtectedBrowserStorage
```
Comment (pranavkm) [inline aspnetcore/blazor/state-management.md:233]:
```suggestion
@using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage
```

File: aspnetcore/blazor/state-management.md [hunk 2]
```diff
@@ -320,7 +311,7 @@ To disable prerendering, open the `Pages/_Host.cshtml` file and change the `rend
 Prerendering might be useful for other pages that don't use `localStorage` or `sessionStorage`. To retain prerendering, defer the loading operation until the browser is connected to the circuit. The following is an example for storing a counter value:
 
 ```razor
-@using Microsoft.AspNetCore.Components.Web.Extensions
+@using Microsoft.AspNetCore.Componets.Server.ProtectedBrowserStorage
```
Comment (pranavkm) [inline aspnetcore/blazor/state-management.md:314]:
```suggestion
@using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage
```

File: aspnetcore/blazor/state-management.md [hunk 3]
```diff
@@ -368,7 +359,7 @@ If many components rely on browser-based storage, re-implementing state provider
 In the following example of a `CounterStateProvider` component, counter data is persisted to `sessionStorage`:
 
 ```razor
-@using Microsoft.AspNetCore.Components.Web.Extensions
+@using Microsoft.AspNetCore.Componets.Server.ProtectedBrowserStorage
```
Comment (pranavkm) [inline aspnetcore/blazor/state-management.md:362]:
```suggestion
@using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage
```

File: aspnetcore/blazor/state-management.md (hunk 1 above)
Comment (pranavkm) [reply aspnetcore/blazor/state-management.md:233]: Should we retitle this section to say 

> ProtectedBrowserStorage in Blazor Server

**Review by pranavkm — APPROVED** [review]

File: aspnetcore/blazor/state-management.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/blazor/state-management.md:233]: It's on a doc "pivot," so they'll select "Blazor Server" at the top and the entire topic will be on Blazor Server.

**Pull request merged**